react-hot-button-component/src/index.jsx

```diff
@@ -18,16 +18,12 @@ function CustomButton() {
   const [count, setCount] = useState(0);
   const handleClick = () => {
     setCount(count + 1);
-  };
-
-  useEffect(() => {
     const button = document.querySelector('button');
```

Comment:
Sorry, but not only do I want the `useEffect` gone, I also want `document.querySelector` gone. You should not use that in React code.